### DISCUSSION TITLE
Using SSLMate API key in CT Logs Discovery Provider

### DISCUSSION THREAD

**Discussion by ghost** [OmniTrustILM/ilm#44] [General]
Hello,
where can I add SSLMate API key for Discovery of CT Logs? I could not find the place for setting the API key in CZERTAINLY nor type it manually in the Discovery creation page, since the field for the API key is list and not textfield.
![image](https://github.com/user-attachments/assets/d2ff27dd-8b17-470b-ab79-f6b13c6cf062)

**Comment by 3keyroman** [comment]
Hello @doloban , you need to create API Key Credential. Then you can select it in the list. When there is no API Key in Credentials, the list is empty.

**Reply by ghost** [reply]
Understood, thank you.
